addons/website_sale_loyalty/models/sale_order.py

```diff
@@ -186,3 +188,32 @@ def _gc_abandoned_coupons(self, *args, **kwargs):
         so_to_reset.applied_coupon_ids = False
         for so in so_to_reset:
             so._update_programs_and_rewards()
+
+    ## OVERRIDE ##
+    def _get_claimable_rewards(self, forced_coupons=None):
```

Comment:
A complete override?
This will also impact backend sales module..
I would rather have a specific website version that also includes linked coupons that are not claimed yet.
(you could make another function that acts as a filter to avoid duplicating code if you wish)